src/util/saveArticles.ts

```diff
@@ -293,24 +296,44 @@ export async function saveArticles(zimCreator: ZimCreator, downloader: Downloade
         curArticle = articleId
         const promises: [string, Promise<Error>][] = []
 
+        const nonPaginatedArticleId = articleDetail.title
```

Comment:
Same as before, a "title" is not an "articleId".

Reply:
I got rid of `nonPaginatedArticleId` because it was confusing indeed.